products/migrations/0001_initial.py

```diff
@@ -0,0 +1,44 @@
+# Generated by Django 4.2.15 on 2024-08-29 09:33
```

Comment:
I think all migrations needs to be added into .gitignore

Reply:
thanks for pointing it out .. ill fix it right away